ServerCore/Pages/Events/Standings.cshtml

```diff
@@ -35,9 +35,14 @@
                     </th>
                     @if (!Model.Event.HideHints)
                     {
+                        <th>
+                            <a asp-page="./Standings" asp-route-sort="@(Model.SortForColumnLink(StandingsModel.SortOrder.HintsEarnedAscending, StandingsModel.SortOrder.HintsEarnedDescending))">
```

Comment:
Is the first entry the default sort when you sort by the column? I'd say descending for HintsEarned makes more sense

Reply:
Agree - fixed this and several others.